### PR TITLE
Fixed Weblate bugs and other translation related issues

### LINKED ISSUE #3025: "seconds" string is not available for translation
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I checked if the issue/feature exists in the latest version.
- [ ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

Hey friends, please add ["seconds"](https://github.com/TeamNewPipe/NewPipe/blob/5bf1df9f1403cbd4de9d9cc7d4f8945ac98ef29e/app/src/main/res/values/settings_keys.xml#L33) string to able translate it. [https://imgur.com/r1wAow8](https://imgur.com/r1wAow8)

 #translation #localization

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
+ Videos in search -> three dots will be translatable in Weblate (it was not translated and not set translatable="false")
+ seek duration (duration for double tap in player) will be translatable in Weblate
+ made default_content_value and default_country_value untranslatable as it should be
+ deleted the strings comments, which was unused and untranslated, to prevent a bug like we have and I just fixed with videos string

Fixes #3025

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -589,4 +587,11 @@
     <string name="downloads_storage_use_saf_title">Use SAF</string>
     <string name="downloads_storage_use_saf_summary">The Storage Access Framework allows downloads to an external SD card.\nNote: some devices are not compatible</string>
     <string name="choose_instance_prompt">Choose an instance</string>
+    <string name="videos_fixed">Videos</string>
```
Comment (TobiGr) [inline app/src/main/res/values/strings.xml]: Why did you rename this?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -589,4 +587,11 @@
...
+    <string name="five_seconds">5 seconds</string>
```
Comment (TobiGr) [inline app/src/main/res/values/strings.xml]: Wouldn't be `%s seconds` better here? We would need to add the seek_duration_description items dynamically, but this shouldn't be a problem.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (B0pol) [reply app/src/main/res/values/strings.xml]: I tried to explain in the first bullet, I'll retry
This string, which is located here in the app [screenshot](https://i.ibb.co/rtxq5yk/Screenshot-20200128-133137-New-Pipe-PR-2309-1.png), is not translated, and also in weblate, it's not available. So by renaming it, it would be automatically added in Weblate.

If you know how to trigger weblate for a specific string, it's unecessary, but I asked on IRC and TheAssasin answered he have no idea.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ocean-sicard-baux) [reply app/src/main/res/values/strings.xml]: I just did it.
The reason why I've chosen this way is: summary is now fine, but when you click on the list, it only shows the value (eg 5, 10, 15…), but not seconds anymore.

Comment (B0pol) [reply app/src/main/res/values/strings.xml]: I firstly done that, but there is a problem: summary is now fine, but when you click on the list, it only shows the value (eg 5, 10, 15…), but not seconds anymore.

 

(I made this with the last commit)

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (TobiGr) [reply app/src/main/res/values/strings.xml]: I think the problem is that another plural string with the same name exists:
https://github.com/TeamNewPipe/NewPipe/blob/8e582aaeddaf8cbed36c920ecff435d24bf5bf39/app/src/main/res/values/strings.xml#L287-L290
`videos_fixed` is a bit misleading. What about `videos_string`?

Comment (B0pol) [reply app/src/main/res/values/strings.xml]: Yeah that's the reason. Also the reason why comments have the same problem. I changed to ``videos_string``.

**Comment by B0pol** [comment]
Closing this pr as commits are screwed up and I don't know how to fix.
Also, as you can see , it's not showing good.
I don't know how to fix this too.
